source/includes/admonitions/note-redact-credentials-command-history.rst

```diff
@@ -0,0 +1,5 @@
+.. note::
+
+   The |mdb-shell| redacts credentials from the :ref:`command history
+   <mdb-shell-command-history>`, but does not redact credentials from 
```

Comment:
Logs do not contain username/password from a provided URL, and instead display `<credentials>`. Commands stored in logs are also run through a redact function to remove user specific information (emails, uris, etc).

So I want to verify if you ran into something that is not being redacted, since that would be a bug. Otherwise command history and logs are on par with what credentials are being hidden.

Reply:
I did run into a bug then. My credentials for a local db were not redacted from the session logs, but were redacted from the command history. I'll file it this morning.

Reply:
EDIT: looks like the credentials aren't being redacted from either in some cases. Filed a bug.